src/Test.FunctionalTests.Sarif/TestData/Multitool/BaselineOption/ExpectedOutputs/TEST1001.ValidateWithBaseline.sarif

```diff
@@ -222,6 +222,7 @@
         {
           "ruleId": "SARIF2005",
           "ruleIndex": 2,
+          "level": "warning",
```

Comment:
>level": "warning", [](http://example.com/codeflow?start=11&length=18)

with this change, many rules will start emitting this, since we are setting the level manually to warning based on the configuration.

Reply:
fixed, this is related to the existing code issue of the ShouldLog() did not consider the default configuration. 
The Reason the existing tests pass is, since it only consider the level at the result node,
when the test codes was having e.g. "note" as the default configuration, the current tests also pass the "note" to explicitly as the level at the result node the same.
Now I have fix the ShouldLog() to also consider the default configuration, I just pass null to the level at the result node.
So now you will see those "level": "note" got deleted instead, in these test files.